scripts/core/image.yml

```diff
@@ -173,10 +173,10 @@ name: $x_image_format_t
 members:
     - type: $x_image_format_layout_t
       name: layout
-      desc: "[in] image format component layout"
+      desc: "[in] image format component layout. Media formats can't be used for $X_IMAGE_TYPE_BUFFER."
```

Comment:
@wdamon-intel : don't know if we should have this guarded with a version or not.

Reply:
- What issue is this addressing?
- Why is this a restriction?
- Could media format layouts ever be supported for buffer types?

Instead of wording this as something that *can't* be done, it should be rephrased to something that *should* or *must* be done. For example, in the description of `layout`, state that "media formats are a type of layout", and under type `type` description, state that "media format layouts are unsupported for $X_IMAGE_TYPE_BUFFER", and (possibly?) wrap it with a version guard...